### DISCUSSION TITLE
Move PP legality issue with pokemon HOME transfers and PLA

### DISCUSSION THREAD

**Discussion by NinFanBoyFTW** [kwsch/PKHeX#3538] [Q&A] [answered]
Hi, I started transferring pokemon from between PLA and pokemon bank, and now all of those pokemon are giving me legality problems with the number of PP for specific movies. I will attach 1 of the pokemon with the issue
[503-01 - Samurott - 478847ECE75E.zip](https://github.com/kwsch/PKHeX/files/8964178/503-01.-.Samurott.-.478847ECE75E.zip)

**Comment by kwsch** [comment, marked as answer]
Working as intended; HOME has a bug where it does not set the correct PP for things transferring into PLA.

It is preferred that PKHeX flag invalid PP values, even though the official app applies them using the wrong PP table.
